gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -197,6 +199,9 @@ private ManagedChannel createSingleChannel() throws IOException {
     if (keepAliveWithoutCalls != null) {
       builder.keepAliveWithoutCalls(keepAliveWithoutCalls);
     }
+    if (interceptorProvider != null) {
```

Comment:
@michaelbausor is this what you mean?

Reply:
Oh! Yes, sorry. Not sure why I didn't parse that as what I was looking for the first time...

Reply:
No worries - thanks for the quick turnaround! Also FYI I don't have write permission on this repo so I can't merge this myself. Let me know if there's anything else I can do.